README.md

```diff
@@ -56,6 +56,11 @@ To get started with WebNN on Intel AI PCs you will need:
 #### Running WebNN on CPU or GPU
 1. Launch Chrome Canary or Edge Canary
 
+* **Notes**:
+
+  1. Running WebNN on Qualcomm GPUs is disabled by default. You can use the `--disable_webnn_for_gpu=0` flag to bypass. However, the functionality is not guaranteed.
```

Comment:
I think it's better for Qualcomm to add such notes, not from Intel. @mingmingtasd

Reply:
I remember we want a README that includes descriptions of all the flags. I can remove the note for
`disable_webnn_for_gpu` for Qualcomm, then that means if people run on Qualcomm and see GPU blocked issue, it's not our responsibility to tell them: you should use `--disable_webnn_for_gpu=0` flag. WDYT?

Reply:
But agree to remove the sentence "However, the functionality is not guaranteed." for Qualcomm, we should not judge other vendors

Reply:
Done by the new commit, PTAL, thanks! @ibelem